google/cloud/documentai_toolbox/wrappers/entity.py

```diff
@@ -57,28 +57,33 @@ class Entity:
     page_offset: dataclasses.InitVar[Optional[int]] = 0
 
     type_: str = dataclasses.field(init=False)
-    mention_text: str = dataclasses.field(init=False, default="")
-    normalized_text: str = dataclasses.field(init=False, default="")
+    mention_text: Optional[str] = dataclasses.field(init=False, default=None)
+    normalized_text: Optional[str] = dataclasses.field(init=False, default=None)
 
-    start_page: int = dataclasses.field(init=False)
-    # Only Populated for Splitter/Classifier Output
-    end_page: int = dataclasses.field(init=False)
+    # Not populated in Classifier output
```

Comment:
Add this to the docstring instead.

Also the docstring needs to be updated for all the changes here such as field type.